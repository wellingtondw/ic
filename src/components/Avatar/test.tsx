import { renderWithTheme } from '../../utils/tests/helpers'

import {Avatar} from '.'

describe('<Avatar />', () => {
  it('should render correctly', () => {
    const { container } = renderWithTheme(<Avatar />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
