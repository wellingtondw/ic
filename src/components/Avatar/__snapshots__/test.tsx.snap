// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> should render correctly 1`] = `
.c0 {
  width: 23.75rem;
  height: 23.75rem;
  border-radius: 50%;
}

<img
  class="c0"
/>
`;
